newIDE/app/src/UI/FlatButton.js

```diff
@@ -22,6 +22,7 @@ type Props = {|
     flexShrink?: 0,
   |},
   target?: '_blank',
+  tabIndex?: number,
```

Comment:
(as per other comment, remove this as not needed - we never want to use anything other than 0 for tabIndex, and this component is already focusable)